ci/test_python_integration.sh

```diff
@@ -11,15 +11,27 @@ EXITCODE=0
 trap "EXITCODE=1" ERR
 set +e
 
-rapids-logger "pytest cuml integration"
+rapids-logger "pytest cuml integration with cudf.pandas"
 ./ci/run_cuml_integration_pytests.sh \
   --numprocesses=8 \
   --dist=worksteal \
   --junitxml="${RAPIDS_TESTS_DIR}/junit-cuml.xml" \
   --cov-config=../../.coveragerc \
   --cov=cuml \
-  --cov-report=xml:"${RAPIDS_COVERAGE_DIR}/cuml-coverage.xml" \
+  --cov-report=xml:"${RAPIDS_COVERAGE_DIR}/cuml-integ-coverage.xml" \
   --cov-report=term
 
+rapids-logger "pytest cuml.accel integration with cudf.pandas"
+
+./ci/run_cuml_accel_integration_pytests.sh \
+  --numprocesses=8 \
+  --dist=worksteal \
+  --junitxml="${RAPIDS_TESTS_DIR}/junit-cuml.xml" \
+  --cov-config=../../.coveragerc \
+  --cov=cuml \
+  --cov-report=xml:"${RAPIDS_COVERAGE_DIR}/cuml-accel-integ-coverage.xml" \
+  --cov-report=term
```

Comment:
What is the thinking around having many `pytest` invocations in one CI job? There is precedent for it, but it feels like it is taking over now. So might be good to reassess, because it makes finding things in the logs quite tricky vs having one CI job per pytest invocation (or just one pytest invocation) https://github.com/rapidsai/cuml/issues/6357 is somewhat related

Reply:
there is a very non-trivial extra resource usage and time needed for each parallel job we add, so we need to be judicious about how many CI jobs we spawn. These additional set of tests is quite quick, spawning a new job would be significantly slower and more resource consuming on GPU CI resources